db/changes.go

```diff
@@ -1029,20 +1060,10 @@ func (col *DatabaseCollectionWithUser) SimpleMultiChangesFeed(ctx context.Contex
 
 				if waitResponse == WaiterClosed {
 					break outer
+				} else if options.ChangesCtx.Err() != nil {
```

Comment:
I think it would be reasonable to move this to a standalone if check prior to line 1061 - I don't think we actually want to continue processing via `break outer`, even though that's how the previous code behaved, as we're just going to eventually check options.ChangesCtx above and error out anyway.